share/completions/rakudo.fish

```diff
@@ -0,0 +1,24 @@
+complete -c rakudo -n __fish_is_first_token -a '(__fish_complete_suffix)'
+complete -c rakudo -n __fish_is_first_token -a - -f -d 'Read program source from STDIN or start REPL if a TTY'
+complete -c rakudo -s c -d 'Check syntax only (runs BEGIN and CHECK blocks)'
+complete -c rakudo -l doc -d 'Extract documentation and print it as text'
+complete -c rakudo -s e -x -d 'One line of program, strict is enabled by default'
+complete -c rakudo -s h -l help -x -d 'Display help text'
+complete -c rakudo -s n -d 'Run program once for each line of input'
+complete -c rakudo -s p -d 'Same as -n, but also print $_ at the end of lines'
+complete -c rakudo -s I -r -d 'Add the path to the module search path'
+complete -c rakudo -s M -r -d 'Load the module prior to running the program'
```

Comment:
the `-c` in `complete -c` is not necessary anymore, I guess we can start by not adding it to new completions?

Reply:
So far I've not required that, because it means that these completions become incompatible with earlier versions. Which isn't a big problem, but when it's just gratuitous stuff like this it feels weird?

This was changed in fish 3.2, which was released in *march*. I think we can wait a bit longer before we break completions for it *on purpose*.

Reply:
Oh yeah, and we had an issue of a user upgrading fish, and then old shell instances would get this error.
So we should wait at least until we can reasonably expect most users to have upgraded.